integrated_channels/moodle/exporters/content_metadata.py

```diff
@@ -37,18 +37,22 @@ def transform_description(self, content_metadata_item):
         base_description = "<a href={enrollment_url}>To edX Course Page</a><br />".format(
             enrollment_url=enrollment_url)
         full_description = content_metadata_item.get('full_description') or None
+        short_description = content_metadata_item.get('short_description') or None
         if full_description and len(full_description + enrollment_url) <= self.LONG_STRING_LIMIT:
             description = "{base_description}{full_description}".format(
                 base_description=base_description,
                 full_description=full_description
             )
-        else:
-            short_description = content_metadata_item.get(
-                'short_description', content_metadata_item.get('title', '')
-            )
+        elif short_description and len(short_description + enrollment_url) <= self.LONG_STRING_LIMIT:
+            short_description = content_metadata_item.get('short_description')
```

Comment:
think you can re use the short_description variable from line 40?

Reply:
oh right. I added the short_description definition up top after having to define it so many times, but then apparently forgot to remove those definitions! 😅